Examples/outputSinDAC.py

```diff
@@ -1,3 +1,8 @@
+from __future__ import division
+from __future__ import print_function
+from builtins import range
+from builtins import object
+from past.utils import old_div
```

Comment:
I can make sure the Python 3 and the Windows updates will be two separate commits, with the Python 3 update being first.